src/features/script-load.js

```diff
@@ -15,10 +15,25 @@ systemJSPrototype.instantiate = function (url, firstParentUrl) {
     return fetch(url).then(function (resp) {
       return resp.text();
     }).then(function (source) {
-      return [[], function(_export) {
+      return [[], function (_export) {
         return {execute: function() {_export('default', JSON.parse(source))}};
       }];
     });
+  } else if (url.substr(-4) === '.css') {
+    return fetch(url).then(function (resp) {
+      return resp.text();
+    }).then(function (source) {
+      return [[], function (_export) {
+        return {
+          execute: function () {
+            // Relies on a Constructable Stylesheet polyfill
+            const stylesheet = new CSSStyleSheet();
+            stylesheet.replaceSync(source);
+            _export('default', stylesheet);
+          }
+        };
+      }];
+    });
```

Comment:
It might be worth having a dynamic module factory now (see if the overall code size decreases?):

```js
function loadDynamicModule (url, createExec) {
  return fetch(url).then(function (res) {
    return res.text();
  }).then(function (source) {
    return [[], function (_export) {
      return { execute: createExec(_export, source) };
    }];
  });
}
```

Reply:
👍 doing this saved ~60 bytes